### PR TITLE
Rename toggle class for Sponsored content

### PR DESCRIPTION
**Pull request by uplne**
Not much to say really

### CODE REVIEW

File: common/app/views/fragments/commercial/paidForMeta.scala.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
 <div class="paidfor-meta">
     <span class="paidfor-meta__label">Paid content</span>
     <div class="paidfor-label paidfor-meta__more has-popup">
-        <button class="u-button-reset paidfor-label__btn popup__toggle" data-toggle="paidfor-popup--capi-component">About @inlineSvg("arrow-down", "icon")</button>
-        <div class="popup is-off paidfor-popup paidfor-popup--capi-component">
+        <button class="u-button-reset paidfor-label__btn popup__toggle" data-toggle="paidfor-popup">About @inlineSvg("arrow-down", "icon")</button>
+        <div class="popup is-off paidfor-popup paidfor-popup">
```
Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/paidForMeta.scala.html]: There's `paidfor-popup` twice, bro

Comment (uplne) [inline common/app/views/fragments/commercial/paidForMeta.scala.html]: Lol

**Comment by Calanthe** [comment]
:+1:

**Comment by regiskuckaertz** [comment]
:+1:

**Pull request merged**